lms/djangoapps/email_marketing/management/commands/remove_user_attribute.py

```diff
@@ -0,0 +1,54 @@
+"""
+Management command to remove provided user var(s) from sail-thru
+"""
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import User
+
+from email_marketing.tasks import update_sailthru_vars
+from util.query import use_read_replica_if_available
+
+
+class Command(BaseCommand):
+    """
+        This command removes custom attributes from user profiles on sailthru
+        Example usage:
+            $ ./manage.py lms remove_user_attribute --attribute=euid --chunk-size=2000
+        OR
+            $ ./manage.py lms remove_user_attribute --attribute=euid,grade
+    """
+
+    help = 'Removes a given user var(s) from sailthru'
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--attribute',
+            type=str,
+            required=True,
+            help='attribute(s) to remove from custom vars from user profile'
+        )
+
+        parser.add_argument(
+            '--chunk-size',
```

Comment:
Having a chunk size here is good, but it would also be good to pair it with a `--delay` value so that we can control load on this.